emcc.py

```diff
@@ -2308,7 +2315,7 @@ def dedup_list(lst):
 
       shared.JS.handle_license(final_js)
 
-      if final_suffix in JS_ENDINGS:
+      if options.oformat in ('js', 'mjs'):
```

Comment:
I think it may be worth keeping `JS_ENDINGS` for this? but I don't feel strongly.

Reply:
except these are not endings .. I guess I should use an enum here really.